### DISCUSSION TITLE
Purpose of directories

### DISCUSSION THREAD

**Discussion by SP-937-215** [simple-login/app#1169] [Q&A]
Hello, 

I don't get the purpose of directories and the documentation is not helpful. Could you give a detailed practical example? 

Also, I understand directory names are unique per domain name and I am pretty sure every one wants a directory called shopping or subscription so if only the happy few get benefits from this feature, what is the purpose? 

Thanks.

**Comment by iGormilhit** [comment]
I have the same questions and here what I think about it: it allows the user to create an alias on the fly without using its username in the domain. But of course, it's mandatory to have unique names by domain.
